src/components/inputs/ErrorMessage/ErrorMessage.js

```diff
@@ -16,4 +16,5 @@ const ErrorMessage = styled.div`
   line-height: 18px;
 `
 
+/** @component */
```

Comment:
I've seen this in other components. What exactly does this declaration do?

Reply:
This is used by the `react-styleguidist` to determine which components it should build a doc page for in the Style Guide, if we don't add this comment it prints out a warning when compiling the project

Reply:
That's awesome! Thanks for explaining that 😄